docs/examples/flex/overflowing.js

```diff
@@ -0,0 +1,25 @@
+// @flow strict
+import { type Node } from 'react';
+import { Flex, Box, Text, Icon, Badge } from 'gestalt';
+
+export default function Example(): Node {
+  return (
+    <Flex width="100%" height="100%" justifyContent="center" alignItems="center">
+      <Box borderStyle="sm" padding={3} rounding={3}>
+        <Flex alignItems="center" gap={3}>
+          <Icon accessibilityLabel="Private" icon="lock" />
+
+          <Flex.Item minWidth={0}>
+            <Text>
```

Comment:
We mention truncation in the description of this section. Should the Text have lineClamp={1} to show that? I don't see any difference in this example when the minWidth={0} is removed. 

Doesn't have to be fixed in this PR, just thought I'd bring it up while we're here

Reply:
DONE